.prettierrc

```diff
@@ -1,5 +1,7 @@
-semi: false
-singleQuote: true
-printWidth: 80
-trailingComma: 'none'
-arrowParens: 'avoid'
+{
+  "semi": false,
+  "singleQuote": true,
+  "printWidth": 80,
+  "trailingComma": "all",
```

Comment:
Why are we making this change?

Reply:
For convenience and to reduce extra changes in the git history.
If we want to add a new property, we needn't add a comma into the last line, but only add a new line.

Reply:
After discussion, we decided to
- work on stylistic PRs that worth merging regularly (like every minor), when most of PRs are merged.
- avoid conflicting useful PRs
- merge PRs to a branch, and then squash merge to main branch, and add the commit to `.git-blame-ignore-revs` file.